products/browsersim-standalone/pom.xml

```diff
@@ -224,46 +260,27 @@
 					</dependency>
 				</dependencies>
 			</plugin>
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>1.7</version>
-				<executions>
-					<execution>
-						<id>attach-standalone</id>
-						<phase>package</phase>
-						<goals>
-							<goal>attach-artifact</goal>
-						</goals>
-						<configuration>
-							<artifacts>
-								<artifact>
-									<file>${project.build.directory}/application/browsersim.jar</file>
-									<type>jar</type>
-									<classifier>standalone</classifier>
-								</artifact>
-							</artifacts>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
+			<!-- JBIDE-JBIDE-18772 do we sill need this extra artfact?
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-assembly-plugin</artifactId>
 				<configuration>
 					<descriptor>assembly/config.xml</descriptor>
-					<!--TODO: Add correct build qualifier if it will not be done during Jenkins build-->
-					<finalName>browsersim-${project.version}-${buildQualifier}</finalName>
 				</configuration>
 				<executions>
 					<execution>
+						<id>fully-qualified</id>
 						<phase>package</phase>
 						<goals>
 							<goal>single</goal>
 						</goals>
+						<configuration>
+							<finalName>browsersim-${project.version}-${buildQualifier}</finalName>
+						</configuration>
 					</execution>
 				</executions>
 			</plugin>
+			-->
```

Comment:
Well, I wanted to ask the browsersim guys if they need the extra artifact for some reason.